test/integration/export_int_test.go

```diff
@@ -65,7 +65,7 @@ func (suite *ExportIntegrationTestSuite) TestExport_ConfigDir() {
 	ts := e2e.New(suite.T(), false)
 	defer ts.Close()
 
-	ts.PrepareProject("cli-integration-tests/Export", "")
+	ts.PrepareProject("cli-integration-tests/Export", "00000000-0000-0000-0000-000000000000")
```

Comment:
If we're not leaving this blank, than maybe we should just always use a valid UUID. It's weird that some tests use a valid UUID and others use this junk one.

Reply:
That's fair. I definitely wanted to go the explicit route.